### DISCUSSION TITLE
Different ssh key and ssh key with passphrase

### DISCUSSION THREAD

**Discussion by supercobra** [basecamp/kamal#406] [Q&A] [answered]
Is there a way to specify a different ssh key in deploy.yml? Is using a key with password possible? How?

**Comment by milushov** [comment]
As far as I know, it's not possible right now

**Comment by supercobra** [comment, marked as answer]
SOLUTION credit @huksley  

you can use standard way of ${HOME}/.ssh/config to designate key for specific host(s)
e.g.

Host test1
  HostName 1.1.1.1
  User test1
  IdentityFile ~/.ssh/id_rsa.other

**Comment by equivalent** [comment]
same answer, just explained for dummies like me :

in `~/.ssh/config`

```
Host 1.22.333.444
  HostName 1.22.333.444
  User root
  IdentityFile ~/.ssh/differentSshKey

```

then 

`$ ssh 1.22.333.444`
👍

also makes `scp` works nicely
